js/phemex.js

```diff
@@ -2769,6 +2770,87 @@ module.exports = class phemex extends Exchange {
         };
     }
 
+    async fetchFundingRate (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (!market['swap']) {
+            throw new BadRequest ('Funding rates only exist for swap contracts');
```

Comment:
Make it `throw new BadSymbol (this.id + ' fetchFundingRate() supports swap contracts only');` to match message on other exchanges